[{"id":893279532,"node_id":"R_kgDONT5dLA","name":"CVE-2024-48990-PoC","full_name":"Cyb3rFr0g/CVE-2024-48990-PoC","private":false,"owner":{"login":"Cyb3rFr0g","id":65209938,"node_id":"MDQ6VXNlcjY1MjA5OTM4","avatar_url":"https://avatars.githubusercontent.com/u/65209938?v=4","url":"https://api.github.com/users/Cyb3rFr0g","html_url":"https://github.com/Cyb3rFr0g","type":"User","site_admin":false},"html_url":"https://github.com/Cyb3rFr0g/CVE-2024-48990-PoC","description":"My take on the needrestart Python CVE-2024-48990","fork":false,"url":"https://api.github.com/repos/Cyb3rFr0g/CVE-2024-48990-PoC","created_at":"2024-11-24T02:12:36Z","updated_at":"2024-11-24T02:32:57Z","pushed_at":"2024-11-24T02:32:54Z","stargazers_count":0,"watchers_count":0,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":[],"visibility":"public","forks":0,"watchers":0,"default_branch":"main","score":1}]